packages/indy-sdk-to-askar-migration/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@aries-framework/indy-sdk-to-askar-migration",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.3.3",
+  "files": [
+    "build"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/indy-sdk-to-askar-migration",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/indy-sdk-to-askar-migration"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aries-framework/anoncreds": "0.4.0-alpha.71",
+    "@aries-framework/askar": "0.4.0-alpha.71",
+    "@aries-framework/core": "0.4.0-alpha.71",
+    "@aries-framework/indy-sdk": "0.4.0-alpha.71",
+    "@aries-framework/node": "0.4.0-alpha.71",
+    "@hyperledger/aries-askar-shared": "file:/Users/beri/Developer/work/hyperledger/aries-askar/wrappers/javascript/aries-askar-shared"
```

Comment:
Will replace the local files when they are released again.

Reply:
Shouldn't the reference to `@aries-framework/*` packages point to * instead of an specific version (0.4.0-alpha.71)?

Reply:
I think this should use 0.3.3 (will be updated when released)

Reply:
Are there any blockers for a new release? Otherwise we can cut one today

Reply:
Yes, the. migration addition. So just releasing the wrappers would be good.